MediaBrowser.Providers/Manager/MetadataService.cs

```diff
@@ -789,11 +798,28 @@ public bool CanRefreshPrimary(Type type)
                 await RunCustomProvider(provider, item, logName, options, refreshResult, cancellationToken).ConfigureAwait(false);
             }
 
-            // ImportUserData(item, userDataList, cancellationToken);
+            ImportUserData(item, userDataList, cancellationToken);
 
             return refreshResult;
         }
 
+        protected virtual void ImportUserData(TItemType item, List<UserItemData> userDataList, CancellationToken cancellationToken)
+        {
+            if (UserManager == null)
+            {
+                return;
+            }
+            if (UserDataManager == null)
+            {
+                return;
+            }
```

Comment:
```suggestion
            if (UserManager == null)
            {
                return;
            }

            if (UserDataManager == null)
            {
                return;
            }

```

linter should be displaying warnings for this

Reply:
Will be changed.